libcloud/dns/drivers/powerdns.py

```diff
@@ -76,7 +78,7 @@ class PowerDNSDriver(DNSDriver):
     RECORD_TYPE_MAP = {
         RecordType.A: 'A',
         RecordType.AAAA: 'AAAA',
-        RecordType.AFSDB: 'AFSDB',
+        # RecordType.AFSDB: 'AFSDB',
```

Comment:
This AFSDB change looks like an error. Please remove it from your PR. PowerDNS supports AFSDB record types.